lib/engine/game/g_1860/game.rb

```diff
@@ -1683,6 +1683,7 @@ def visit_route(ridx, intersects, visited)
         # all routes must intersect each other
         def check_intersection(routes)
           actual_routes = routes.reject { |r| r.chains.empty? }
+          return unless actual_routes
```

Comment:
actual_routes will never be false, arrays are always truthy

Reply:
Jinx. You owe me a soda.